### PR TITLE
Added columns

### PR DESCRIPTION
**Pull request by lorienhall**

### CODE REVIEW

**Review by LtShibby — COMMENTED** [review]
make sure all <li>s are closed

File: docs/index.html [hunk 1]
```diff
@@ -15,23 +15,45 @@
...
   		    padding: 5px;
   		    border: 2px solid;
 	    }
 	    
 	    body, h2 {
-	        padding-top: 65px; 
+	        padding-top: 65px;
 	    }
 	    
+	    table {
```
Comment (Mbd06b) [inline docs/index.html:27]: Don't use html tag selectors in your custom css.  Create a class that describes your change and apply that class to the element. 

Use a class selector name like " .solid-table-border"  and apply the class to the table element with class="solid-table-border" attribute.

File: docs/index.html [hunk 2]
```diff
@@ -15,23 +15,45 @@
...
 	    .fixed {
   		    position: fixed;
-            top: 0;
-            width: 100%;
+  		    top: 0;
   		    padding: 5px;
   		    border: 2px solid;
 	    }
 	    
 	    body, h2 {
```
Comment (Mbd06b) [inline docs/index.html:23]: Don't use html tag selectors in your custom css.  Create a class that describes your change and apply that class to the element. 
Even better, use bootstrap classes to the HTML elements to apply the desired spacing.  https://getbootstrap.com/docs/4.0/utilities/spacing/

**Comment by lorienhall** [comment]
According to the html checker, everything is closed. Did you see something specific

File: docs/index.html (hunk 1 above)
Comment (lorienhall) [reply docs/index.html:27]: Should I move the css classes into the css file?

**Review by LtShibby — APPROVED** [review]

**Pull request merged**